docs/misc/declarative-configuration/config.yaml

```diff
@@ -0,0 +1,63 @@
+# This file serves as a specification how to map properties from the existing config properties
+# to the declarative configuration format.
+# This list does not contain all properties, but those that
+# are different from what the config bridge already does.
+
+# Based on https://github.com/open-telemetry/opentelemetry-configuration/blob/main/examples/kitchen-sink.yaml
```

Comment:
I think we should guide ultimately guide users to use some version of https://github.com/open-telemetry/opentelemetry-configuration/blob/main/examples/sdk-migration-config.yaml and / or https://github.com/open-telemetry/opentelemetry-configuration/blob/main/examples/sdk-config.yaml, potentially with java agent specific config as you've shown here. 

The `kitchen-sink.yaml` is meant to show all all the different configuration surface area, but isn't representative of what most users will want to do.